etc/utils/src/logs.ts

```diff
@@ -1,7 +1,7 @@
 import path from 'path';
 import fs from 'node:fs/promises';
 
-const pathToHome = path.join(__dirname, '../../../..');
+const pathToHome = path.join(__dirname, '../../..');
```

Comment:
This may break something in CI or zk_toolbox, but the previous location looks incorrect; it's the *parent* directory for the Era repo, not the repo directory itself.